counsel.el

```diff
@@ -4562,8 +4570,12 @@ If there is no such buffer, start a new `shell' with NAME."
                              tag)
                            (split-string (match-string 1 a) "," t)
                            ":"))))
-          (push (cons (if tags (concat text " :" tags ":") text)
-                      href)
+          (push (list
+                 (concat (if tags (concat name " :" tags ":") name)
+                         " "
+                         (propertize href 'face 'counsel-firefox-bookmarks-location))
```

Comment:
Please use `put-text-property` instead of `propertize` to avoid unnecessary string allocation.

Reply:
done